bangazonapi/views/product.py

```diff
@@ -281,6 +282,8 @@ def price_filter(product):
                 return False
             products = filter(price_filter, products)
 
+        if product_location is not None:
+            products = products.filter(location__contains=product_location)
```

Comment:
These changes are related to ticket #14 , and should be removed from this branch/PR.